FirebasePerformance/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Pending
```

Comment:
nit: use "Unreleased" so the release tooling can detect and replace it with the next version
```suggestion
# Unreleased
```